dev/tools/grunt/tasks/static.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (grunt) {
         };
 
     grunt.registerTask('static', function (target) {
-        var currentTarget = target || 'test',
+        var currentTarget = target || 'file',
```

Comment:
But... the whole point is to not use `test` on CI, b/c output is saved only as JUnit XML file, which isn't available, b/c build artifacts are not saved anywhere (even if they will be available, it's way more complicated way than just read the tests output).

Also, `quiet` option do more harm than good, b/c ignore all warnings. 

Results of `gulp static:test`
```
unning "eslint:test" (eslint) task
Warning: Task "eslint:test" failed. Used --force, continuing.
```

Results of `gulp static` aka. `gulp static:file`
```
/Users/igloczek/Sites/magento2-dev/lib/web/mage/accordion.js
  7:9  error  Parsing error: Unexpected token 'jquery'

✖ 1 problem (1 error, 0 warnings)

Warning: Task "eslint:file" failed. Used --force, continuing.
```